### PR TITLE
[FR] Remove links to 'Deprecated' / 'Feedback Appreciated' terms

### PR DESCRIPTION
**Pull request by guillaumebernard84**
### Describe your changes
Following this PR, doing the same for the not merged yet FR branch : https://github.com/cncf/glossary/pull/2198
Only one occurence found

### Related issue number or link (ex: `resolves #issue-number`)
https://github.com/cncf/glossary/pull/2198

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by sestegra — APPROVED** [review]
LGTM 🚀

File: content/fr/containerization.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ tags: ["application", "", ""]
 
 ## Ce que c'est
 
-La contenérisation est l'action de regrouper une application et ses dépendances dans une [image de conteneur](/container-image/).
+La contenérisation est l'action de regrouper une application et ses dépendances dans une image de conteneur.
```
Comment (huats) [inline content/fr/containerization.md:10]: Finalement on garde avec les accents ? Je crois que nous sommes tous plutôt OK pour ne pas en mettre...

Comment (guillaumebernard84) [reply content/fr/containerization.md]: Haha, tu a raison, j'ai juste cherché les liens, j'ai pas regardé le reste, je corrige

**Comment by sestegra** [comment]
Il me semble que les modifications sont correctes.

**Review by huats — APPROVED** [review]

**Pull request merged**